SQL/0000-00-00-schema.sql

```diff
@@ -695,10 +695,13 @@ DROP TABLE IF EXISTS `notification_spool`;
 CREATE TABLE `notification_spool` (
   `NotificationID` int(11) NOT NULL auto_increment,
   `NotificationTypeID` int(11) NOT NULL default '0',
-  `TimeSpooled` int(11) NOT NULL default '0',
+  `ProcessID` int(11) NOT NULL DEFAULT '0',
+  `TimeSpooled` datetime DEFAULT NULL,
```

Comment:
The MRI-pipeline is now tested and another pull request is created in regards to this modification:
https://github.com/aces/Loris-MRI/pull/70